corehq/apps/hqcase/api/core.py

```diff
@@ -20,6 +21,7 @@ def serialize_case(case):
         "indexed_on": _isoformat(datetime.utcnow()),
         "closed": case.closed,
         "date_closed": _isoformat(case.closed_on),
+        "is_deprecated": is_case_type_deprecated(case.domain, case.type),
```

Comment:
That's a good point - I wonder if denormalizing this is appropriate.  Being deprecated is an attribute of the case type, _not_ of individual cases.  Take someone using the API to populate a SQL DB (using the DET, for instance) - after deprecating a case type, only cases pulled _after_ deprecation will be flagged as deprecated, while cases pulled before that action won't be marked as such, even though conceptually they are now deprecated.

Reply:
@esoergel Given your example, do you think we should then rather not include the `is_deprecated` property in the API case response? Given that this has no real effect on cases themselves, asides from simply serving as a potential warning flag, it might then be better to not have this property included.

Reply:
Yeah, I agree with that.  I think the ideal place for it would be in a "case types" API (or data dictionary API), if one existed.

Reply:
I have removed the `is_deprecated` property from the case API response ([256d7c9](https://github.com/dimagi/commcare-hq/pull/33089/commits/256d7c9cb3b77ca1c511ebab304b556dc1b53ca4)).

I like the idea of a data dictionary API, where you could pull and update case types and their respective properties. I think especially given that the Data Dictionary is being GA'ed, this would be a useful feature to have.